README.md

```diff
@@ -11,6 +11,20 @@ Summary
 -----------
 This project is a .NET implementation of the [Apache Kafka] protocol.  The wire protocol portion is based on the [kafka-python] library writen by [David Arthur] and the general class layout attempts to follow a similar pattern as his project.  To that end, this project builds up from the low level KafkaConnection object for handling async requests to/from the kafka server, all the way up to a higher level Producer/Consumer classes.
```

Comment:
Need to say in the beginning that this is a fork of Jroland's kafka-net library.. suggested order:
This library is a forn of Jroland's kafka-net library, with adjustments and improvements.
The original project is a .NET implementation of the ... (the original paragraph).

Improvements and changes: (your list)